### ISSUE TITLE
Implement a call to update Core Exchane Rate (and ONLY CER) for UIA

### ISSUE THREAD

**Issue by DEEXPrivate** [bitshares/python-bitshares#82]
At the moment there is only Bitshares.publish_price_feed() method that works only with market pegged assets. Having the ability to update only CER from the library would be quite useful :)

**Comment by DEEXPrivate** [comment]
Aren't there an 'asset_update' operation that is supposed to be able to do that?
Using it for that purpose will, of course, require resetting asset options (if i am reading source code correctly, that is), but can't a library method that hides away the complexity be implemented in python?

**Comment by abitmore** [comment]
Yes, you're correct on this. I was talking about another thing: enable CER feeds for all assets but not only bitAssets.
